tests/dnslink_gateway_test.go

```diff
@@ -13,134 +13,11 @@ import (
 	"github.com/ipfs/gateway-conformance/tooling/tmpl"
 )
 
-// TODO(laurent): this were in t0114_gateway_subdomains_test.go
-
-func TestDNSLinkGateway(t *testing.T) {
-	tests := SugarTests{
-		// ## ============================================================================
-		// ## Test DNSLink requests with a custom PublicGateway (hostname config)
-		// ## (DNSLink site at http://dnslink-test.example.com)
-		// ## ============================================================================
-		// # disable wildcard DNSLink gateway
-		// # and enable it on specific NSLink hostname
-		// ipfs config --json Gateway.NoDNSLink true && \
-		// ipfs config --json Gateway.PublicGateways '{
-		//   "dnslink-enabled-on-fqdn.example.org": {
-		//     "NoDNSLink": false,
-		//     "UseSubdomains": false,
-		//     "Paths": ["/ipfs"]
-		//   },
-		//   "only-dnslink-enabled-on-fqdn.example.org": {
-		//     "NoDNSLink": false,
-		//     "UseSubdomains": false,
-		//     "Paths": []
-		//   },
-		//   "dnslink-disabled-on-fqdn.example.com": {
-		//     "NoDNSLink": true,
-		//     "UseSubdomains": false,
-		//     "Paths": []
-		//   }
-		// }' || exit 1
```

Comment:
highlight: test here is focused on differences between different Kubo configurations,
I did not port it.